examples/src/main/java/org/apache/ignite/examples/client/ClientKubernetesPutGetExample.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.examples.client;
+
+import org.apache.ignite.Ignition;
+import org.apache.ignite.client.ClientCache;
+import org.apache.ignite.client.IgniteClient;
+import org.apache.ignite.client.ThinClientKubernetesAddressFinder;
+import org.apache.ignite.configuration.ClientConfiguration;
+import org.apache.ignite.examples.model.Address;
+import org.apache.ignite.kubernetes.configuration.KubernetesConnectionConfiguration;
+
+/**
+ * Demonstrates how to use Ignite thin client within the Kubernetes cluster using KubernetesConnectionConfiguration.
```

Comment:
@Nikita-tech-writer, please check the comments in the source code files too. I see some spellings below.

Reply:
@dmagda OK, sure